assets/components/svg/svg.jsx

```diff
@@ -82,6 +82,35 @@ function SvgExclamation() {
 
 }
 
+// Alternate error exclamation mark, used on Direct Debit pop up.
```

Comment:
Can this one be the same of `SvgExclamation` so in that way we re-use components?

Reply:
Following the convetion introduced by @Ap0c, something like 
```js
 // ----- Auxiliary components ----- //
```
would work.

Reply:
Sadly the design calls for a slightly different exclamation mark! Even if it was the same, it takes as much CSS to restyle it (this has a border and is red on a white background instead of white on a red background) as just including a new SVG, so I think this way is reasonable